.circleci/verdaccio/config.yml

```diff
@@ -0,0 +1,27 @@
+storage: ./storage
+auth:
+  htpasswd:
+    file: ./htpasswd
+uplinks:
+  npmjs:
+    url: https://registry.yarnpkg.com/
+    max_fails: 40
+    maxage: 30m
+    timeout: 60s
+    fail_timeout: 10m
+    cache: false
+    agent_options:
+      keepAlive: true
+      maxSockets: 40
+      maxFreeSockets: 10
+packages:
+  '@*/*':
+    access: $all
+    publish: $all
+    proxy: npmjs
+  '**':
+    access: $all
+    publish: $all
+    proxy: npmjs
+logs:
+  - {type: file, path: verdaccio.log, format: pretty, level: debug}
```

Comment:
Debug is really verbose, I'd recommend `warn` and also `pretty` is pretty expensive in terms of execution, if you don't need the coolers or fancy arrows, you might consider `json` instead to boost the registry.